lib/LMSManagers/LMSHelpdeskManager.php

```diff
@@ -97,6 +97,16 @@ public function GetQueueContents($ids, $order = 'createtime,desc', $state = NULL
 		elseif (empty($priority))
 			$priorityfilter = '';
 
+                if (isset($netdevids) && is_array($netdevids))
+                        $netdevidsfilter = ' AND t.netdevid IN (' . implode(',', $netdevids) . ')';
+                elseif (empty($netdevids))
+                        $netdevidsfilter = '';
+
+                if (isset($netnodeids) && is_array($netnodeids))
```

Comment:
Czy może wystąpić sytuacja w której $netnodeds będzie array()-em nie zawierającym w ogóle elementów?

Reply:
@chilek To może dorzucić warunek && !is_null($netdevids) ?
Nie widzę takiej możliwości. W pozostałych warunkach jest tak samo.

Reply:
!is_null($netdevids) nie ma sensu, bo to już isset() załatwia.
Może !empty($netdevids)? Wtedy można zrezygnować z isset($netdevids).

Reply:
Poprawiłem we wszystkich warunkach.